integration_test/third_party_apps_data/agent/ops-agent/linux/enable_cassandra

```diff
@@ -0,0 +1,46 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+logging:
+  receivers:
+    cassandra_system:
+      type: cassandra_system
+      include_paths:
```

Comment:
Do we have to explicitly configure `include_paths` here? If `/home/test_user/apache-cassandra/logs/XXX` paths are used by default, can we just add them to the default paths? Or is that just something we use for testing (I could not find its reference in the testing bits though)

Reply:
That's specifically set up for testing SLES where there's no default package install, it's just a manual download, unzip and run. Since kokoro only validates against Debian right now I haven't fully validated that it works in there but it's designed for kokoro testing, not for a default install path.

Reply:
Ah I see. Can we add a comment here to clarify why we need to explicitly specify `include_paths`, or customize the Cassandra logs on SLES to write to `/var/log/cassandra/XXX`?

Reply:
Can you do the SLES case as a separate receiver, so that we can still test the built-in default paths on Debian/Ubuntu/RedHat?